src/nxdoc/web-ui/web-ui-release-notes/web-ui-release-notes-3-1-0.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Version 3.1.0
+description: Discover what's new in Web UI 3.1.0.
+review:
+    comment: ''
+    date: '2023-03-28'
+    status: ok
+toc: true
+labels:
+tree_item_index: 982
+hidden: true
+---
+
+{{{multiexcerpt 'matching-notes' page='web-ui-release-notes'}}}
+
+{{! multiexcerpt name='web-ui-updates'}}
+## What’s New in Web UI for LTS 2021 (Version 3.1.0)
+
+This release doesn't include any release tickets.
+
+### Other Noteworthy Changes
+
+Don't have any release tickets in this Release.
```

Comment:
```suggestion
Compared to Web UI 3.0.[insert release number here], this release provides the following additional bugfixes.
```
We need to mention bugfixes that were done since the latest LTS 2021 release here if there are any. If there's nothing and both releases are exactly the same, we should mention it instead.